kitchen.yml

```diff
@@ -130,8 +124,11 @@ suites:
     includes:
       - ubuntu-18.04
       - ubuntu-16.04
+      - debian-9
       - centos-7
-  - name: certbot
+      - oracle-7.6
+      - oracle-6.10
+  - name: certbot # This is impossible to do in kitchen, so this is a list of platforms that have been tested elsewhere
```

Comment:
Should probably document why incase it changes in the future

Reply:
Added clarifying comment.